build.sbt

```diff
@@ -467,6 +470,7 @@ lazy val testkit = pekkoModule("testkit")
   .settings(AutomaticModuleName.settings("pekko.actor.testkit"))
   .settings(OSGi.testkit)
   .settings(initialCommands += "import org.apache.pekko.testkit._")
+  .enablePlugins(SbtOsgi)
```

Comment:
Does the tests need osgi too？

Reply:
It's inherited from the parent project. Note that if it's  OT done correctly the sbt build won't even load.

Reply:
Also the tests should not have OSGi enabled anyways, OSGi is about packaging the final jar and we don't package tests

Reply:
Also the tests should not have OSGi enabled anyways, OSGi is about packaging the final jar and we don't package tests